ddp-workspace/projects/ddp-pancan/src/app/components/activity/activity.component.ts

```diff
@@ -29,9 +66,13 @@ export class ActivityComponent extends ActivityRedesignedComponent {
     }
 
     get hasNoEmptyNestedActivity(): boolean {
-        const enabledActivityBlocks = this.currentSection.blocks.filter(this.isActiveModalActivityBlock) as ActivityActivityBlock[];
+        try {
+            const enabledActivityBlocks = this.currentSection.blocks.filter(this.isActiveModalActivityBlock) as ActivityActivityBlock[];
 
-        return enabledActivityBlocks.every(this.blockHasAnswers);
+            return enabledActivityBlocks.every(this.blockHasAnswers);
+        } catch {
```

Comment:
Just wondering what kind of error can be caught there ?

Reply:
It's an error referencing `this.currentSection` causes at times, I haven't investigated the root, but I'll look into it

Reply:
Using try/catch here is not a nice idea, imo.
What about just checking if `this.currentSection` exists ? (even though I can't see a case when it does not)

Reply:
The issue was coming from the base class and I've fixed it

Reply:
In this case using try/catch is  a hiding of that issue.
 Like: 
```
 something went wrong (potentially with `this.currentSection`) 
   -> never mind, let's decide that all blocks in the current section have answers 
     -> toggle showEmptyNestedActivityError to false in order to remove an error message
```
     
At least the issue should be logged as to fix it later. 

> The issue was coming from the base class 

It is strange. `currentSection` from the base class is used in many places and never failed before. May be it is reason to create a bug ticket as to investigate it more ?